lib/Agrammon/Email.pm6

```diff
@@ -0,0 +1,70 @@
+use v6;
+use Net::SMTP::Client::Async;
+use MIME::Base64;
+use Email::MIME;
+
+class Agrammon::Email {
+    has $!msg;
+    has $!subject;
+    has $!to;
+    has $!from;
+    has $!attachment;
+    has $!mail;
+
+    submethod TWEAK( :$attachment, :$msg, :$filename, :$!to, :$!from, :$!subject --> Nil) {
```

Comment:
Hm, looking at this again, I wonder if some of these should be marked required to ensure correct usage (as nameds are all optional by default, but it looks like only the attachment is really optional).

Reply:
Fixed. Gave the filename a default.